test/Fortran/fixture-mod/SConstruct

```diff
@@ -0,0 +1,16 @@
+import sys
+
+if sys.platform == "win32":
+    tool_list = ['mingw', 'gfortran', 'gnulink']
+else:
+    tool_list = ['default']
+
+env = Environment(
+    tools=tool_list,
+    F90='gfortran',
```

Comment:
Then remove F90 and LINK being explicitly set?
(On macports its gfortran-mp-## which I can set in the Fortran module, and/or SCons.Platform.darwin